new/lib/validations.tsx

```diff
@@ -0,0 +1,39 @@
+interface ValidationsMap {
+    [key: string]: (input: any, param?: number) => boolean | string;
+}
+export const validationsMap: ValidationsMap = {
+    //Check required
+    "required": (input: any) => {
+        if (input === undefined || input === null || input === "") {
```

Comment:
if the input contains space only, should it be considered as empty string as well?